lib/libs/email/content/email-components.tsx

```diff
@@ -33,7 +33,6 @@ const Textarea = ({ children }: { children: React.ReactNode }) => (
       backgroundColor: "transparent",
       fontSize: "14px",
       lineHeight: "1.4",
-      boxShadow: "0 1px 2px 0 rgba(0, 0, 0, 0.05)",
```

Comment:
Is this important? It was making a weird shadow for the summary section

Reply:
I saw that too, I am for deleting it thanks!